Commands/trashwhat.js

```diff
@@ -0,0 +1,15 @@
+const voice = require("@discordjs/voice");
+
+class TrashWhat {
+    run(client, message, args) {
+        const player = require('../Utility/playvoice');
+        player.Play(message, voice.createAudioResource('Assets/what.m4a'));
+    }
+
+    config = {
+        name: "what",
+        help: "It's not fault, I swear"
```

Comment:
It's not correct, I swear

Reply:
Oops, I will fix that

Reply:
Fixed in ff3bb955578bb7f864ab101f547a2e06a7a04ed4